test/source/tests/setup.ts

```diff
@@ -496,6 +497,84 @@ AN8G3r5Htj8olot+jm9mIa5XLXWzMNUZgg==
       await gmailPage.notPresent(['@webmail-notification', '@notification-setup-action-open-settings', '@notification-setup-action-dismiss', '@notification-setup-action-close']);
     }));
 
+    ava.default('setup [not using key manager] - notify users when their keys expire soon', testWithBrowser(undefined, async (t, browser) => {
+      const acctEmail = 'flowcrypt.notify.expiring.keys@gmail.com';
+      const passphrase = '1234';
+      const warningMsg = 'Your keys are expiring in 18 days. Please import a newer set of keys to use.';
+      const settingsPage = await BrowserRecipe.openSettingsLoginApprove(t, browser, acctEmail);
+      // Generate key that expires in 20 days
+      const key = await opgp.generateKey({
+        curve: 'curve25519',
+        userIds: [{ email: acctEmail }],
+        keyExpirationTime: 20 * 24 * 60 * 60,
+        passphrase,
+        date: new Date(Date.now() - 2 * 24 * 60 * 60 * 1000)
+      });
+      // Setup with above key
+      await SetupPageRecipe.manualEnter(settingsPage, 'unused', {
+        submitPubkey: false,
+        usedPgpBefore: false,
+        key: {
+          title: '?',
+          armored: key.privateKeyArmored,
+          passphrase,
+          longid: '1b383d0334e38b28',
```

Comment:
You just generated a new key and supplying a longid from a totally different key?
If `longid` is not needed, we'd better make it obvious, like
```
longid: '0000000000000000', // dummy -- not needed
```